spec/mysql2/statement_spec.rb

```diff
@@ -689,9 +691,9 @@
   context 'close' do
     it 'should free server resources' do
       stmt = @client.prepare 'SELECT 1'
-      expect { stmt.close }.to change {
-        @client.query("SHOW STATUS LIKE 'Prepared_stmt_count'").first['Value'].to_i
-      }.by(-1)
+      GC.disable
```

Comment:
Sometimes this fails with "expected to change by -1, but changed by -2" but this patch didn't fix it =/